lib/blacklight/maps/maps_search_builder.rb

```diff
@@ -0,0 +1,20 @@
+module BlacklightMaps
+  module MapsSearchBuilder
```

Comment:
Gotcha... I see what you are saying now. I guess then stick with the module pattern. Good to :shipit: then?